Cody

WoollyCoyote opened the pull request:
list of nouns and adjectives, one of each is randomly selected. and combined.  Then inserted into the sname(screen name) input element.

NBryant83 committed:
moved to pages folder

NBryant83 committed:
moved registration form files to pages folder

NBryant83 committed:
completed very basic form styling

NBryant83 committed:
added handleSubmit and handleChange

NBryant83 committed:
minor change to registrationForm.css to push

NBryant83 committed:
imported useEffect

NBryant83 committed:
deleted useEffect

NBryant83 committed:
added screenNameGenerator folder and files

NBryant83 committed:
Merge branch 'nakisha' of github.com:BIRDZ01/Frontend_React into nakisha

WoollyCoyote committed:
added a name generator

added a name generator component to the Registration Form. Generates a random name from a list of birds and list of adjectives and inserts it to the Screen name input.

WoollyCoyote committed:
Update NameGenerator.jsx

cleaned up some non bird nouns that slipped through.

WoollyCoyote committed:
not sure if I pushed the old

updated...maybe

NBryant83 reviewed (approved):
Screen name generator working beautifully! Thanks Cody!

WoollyCoyote committed:
added base components for a upon login page

added a loggin loggout button to navigate app,js
own page component
dailycheck  component
post component

WoollyCoyote committed:
added post logic

added logic to the post, so it add as many post as there are stored, also added post form to give them shape.

WoollyCoyote committed:
updated

post form was not utilized yet, removed to keep clean.

NBryant83 committed:
added emergency contact email to form

WoollyCoyote committed:
Profile pic

profile pic style
default profile pic default generator
lacks saving
